.github/workflows/test.yml

```diff
@@ -0,0 +1,41 @@
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
```

Comment:
```suggestion
    runs-on: ubuntu-20.04
```
I would suggest using ubuntu-20.04 instead, so you don't have a later update to Ubuntu 22.04 break something randomly.